usecases/MinimumWorkingExample/dodo.py

```diff
@@ -374,3 +385,84 @@ def task_upload_to_openbis():
             'targets': [sample_file_path],
             'clean': [clean_targets],
         }
+
+@create_after(executed='extract_metadata_emodul')
+def task_perform_calibration():
+    """Loop over the experiments and store the calibrated E in csv file. Each iteration generates four files
+    viz. calibrated data.csv, displacement data used, load data used and the knowledge graph file containing the calibration details
+    (these are not specific file names)"""
+    # create folder, if it is not there
+    Path(calibrated_data_directory).mkdir(parents=True, exist_ok=True)
+
+    # defining calibration input, setting the values for the priors
+    E_loc = 30  # KN/mm2 (mean)
+    E_scale = 10  # KN/mm2 (std)
+
+    # setting for fast test, defining the list
+    if config['mode'] == 'cheap':
+        list_exp_name = [single_example_name]
+    else:  # go through all files
+        list_exp_name = os.listdir(
+            processed_data_emodulus_directory)
+        list_exp_name = [os.path.splitext(f)[0] for f in list_exp_name]  # split extension
+    for f in list_exp_name:
+        exp_metadata_path = os.path.join(metadata_emodulus_directory, f + '.yaml')
+        with open(exp_metadata_path) as file:
+            data = yaml.safe_load(file)
+        diameter = float(data['diameter'])
+        length = float(data['length'])
```

Comment:
the length is not required for the calculation, as only the gauge length is simulated, which is constant.

Reply:
Ah you mean even if I dont pass the length, the calibration will work just right? I do not remember what was gauge length.

Reply:
Yes the actual length is not required for our example. The gauge length (the length over which is actually measured) is currently set here: https://github.com/BAMresearch/LebeDigital/blob/e751437b36c89f3960a01b5116cb72a4f2717c19/lebedigital/calibration/forwardmodel_linear_elastic_cylinder.py#L44
After looking at it, I would suggest to change line 413 in the dodo to "length = 100"  (or whatever units we are using) and then carry the value all the way to line 44 in the forwardmodel_linear_elastic_cylinder.py. This way we are still flexible. I would suggest to Stephan to inlcude the gauge length in the metadata. @joergfunger: do you agree?

Reply:
updated